CHANGES.rst

```diff
@@ -22,14 +22,29 @@ Bug fixes:
 - *add item here*
 
 
+
 5.0.9 (2017-08-05)
 ------------------
 
 New features:
 
+- Add js-shortcuts package for keyboard shortcuts
+  [b4oshany]
+
+- Update package version to 5021 from 5020.
```

Comment:
This is already in master. No need to be in changelog https://github.com/plone/Products.CMFPlone/commit/7da569be37633a798075270d8f0293c207580e68